app/javascript/modules/listingDetailsAside/ListingDetailsAside.scss

```diff
@@ -0,0 +1,20 @@
+.fcfs-no-lottery {
+    --card-border-radius: rounded-none;
+    --card-background-color: var(--seeds-color-gray-200);
+    --card-footer-background-color: var(--seeds-color-gray-200);
+    --card-border-width: var(--seeds-border-0);
+
```

Comment:
If you didn't want to add the extra div around the Card, you could also add the style 
`border-bottom: 1px solid var(--seeds-border-color)` here